flow-defs/desktop-container-type-context.js.flow

```diff
@@ -0,0 +1,7 @@
+// @flow
+
+import * as React from "react";
+export type ContainerType = "small" | "medium" | "large";
```

Comment:
This only refers to desktop, right ?

Reply:
If yes, I would rename to DesktopContainerType

Reply:
yes, you're right, I'll rename it

Reply:
done